tests/typowalking/wordwalker.js

```diff
@@ -27,7 +28,8 @@ bender.test( {
 		wordwalker = new editor.plugins.nanospell.WordWalker(range);
 
 		while (currWordObj = wordwalker.getNextWord()) {
-			wordsReturned.push(currWordObj.word);
+			word = currWordObj.word;
+			if (word) wordsReturned.push(word);
```

Comment:
Hmm.

Maybe something like... set the editor html to get the issue (wordwalker stumbles on an empty word) -> invoke wordwalker on the range a few times and see what comes back?